src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/JsonSerializerWrapper.Reflection.cs

```diff
@@ -881,5 +884,91 @@ private int ReadExactlyFromSource(byte[] buffer, int offset, int count)
             public override void SetLength(long value) => throw new NotSupportedException();
             public override void Write(byte[] buffer, int offset, int count) => throw new NotSupportedException();
         }
+
+        // TODO: Deserialize to use PipeReader overloads once implemented
+        private class PipelinesSerializerWrapper : JsonSerializerWrapper
```

Comment:
Following existing naming convention of `AsyncStreamSerializerWrapper` and `SyncStreamSerializerWrapper`:
```suggestion
        private class AsyncPipelinesSerializerWrapper : JsonSerializerWrapper
```